code/datums/brain_damage/mild.dm

```diff
@@ -162,7 +162,7 @@
 			trigger_mobs = list(/mob/living/simple_animal/hostile/lizard) //they're hostile! of course they're scary!
 			trigger_species = list(/datum/species/lizard)
 		if("spooky skeletons")
-			trigger_words = list("skeleton", "rattle me bones", "milk", "xylophone", "bone", "calcium", "i want to get off mr bones wild ride", "the ride never ends")
+			trigger_words = list("skeleton", "rattle me bones", "milk", "xylophone", "bone", "calcium", "mr bones wild ride", "the ride never ends")
```

Comment:
doesn't the word "bone" make "rattle me bones" and "mr bones wild ride" redundant?

Reply:
True, i gotta review goofcode before merging it